src/functional/ffront/past_to_itir.py

```diff
@@ -226,20 +280,25 @@ def _compute_field_slice(node: past.Subscript):
         return out_field_slice_
 
     def _visit_stencil_call_out_arg(
-        self, node: past.Expr, **kwargs
+        self, node_out: past.Expr, node_field_domain: past.Dict, **kwargs
```

Comment:
This names should be more expressive (calling the out_arg `node` was not nice in the original code) and prepending node doesn't convey extra information in my opinion.
```suggestion
        self, out_arg: past.Expr, domain_arg: past.Dict, **kwargs
```